ansible/site.yml

```diff
@@ -133,3 +133,16 @@
   tasks:
     - import_role:
         name: telemetry-installer
+
+- name: Install Orchestration Manager
```

Comment:
Could u make this module as optional so that users can choose if to deploy it?

Reply:
Done. Thank you.